blockchain/fullblocktests/error.go

```diff
@@ -0,0 +1,308 @@
+
+	// ErrSpendTooHigh indicates a transaction is attempting to spend more value
+	// than the sum of all of its inputs.
+	ErrSpendTooHigh = ErrorKind("ErrSpendTooHigh")
+
+	// ErrTooManySigOps indicates the total number of signature operations for a
+	// transaction or block exceed the maximum allowed limits.
+	ErrTooManySigOps = ErrorKind("ErrTooManySigOps")
+
+	// ErrFirstTxNotCoinbase indicates the first transaction in a block is not a
+	// coinbase transaction.
+	ErrFirstTxNotCoinbase = ErrorKind("ErrFirstTxNotCoinbase")
+
+	// ErrCoinbaseHeight indicates that the encoded height in the coinbase is
+	// incorrect.
+	ErrCoinbaseHeight = ErrorKind("ErrCoinbaseHeight")
+
+	// ErrMultipleCoinbases indicates a block contains more than one coinbase
+	// transaction.
+	ErrMultipleCoinbases = ErrorKind("ErrMultipleCoinbases")
+
+	// ErrStakeTxInRegularTree indicates a stake transaction was found in the
+	// regular transaction tree.
+	ErrStakeTxInRegularTree = ErrorKind("ErrStakeTxInRegularTree")
+
+	// ErrRegTxInStakeTree indicates that a regular transaction was found in the
+	// stake transaction tree.
+	ErrRegTxInStakeTree = ErrorKind("ErrRegTxInStakeTree")
+
+	// ErrBadCoinbaseScriptLen indicates the length of the signature script for
+	// a coinbase transaction is not within the valid range.
+	ErrBadCoinbaseScriptLen = ErrorKind("ErrBadCoinbaseScriptLen")
+
+	// ErrBadCoinbaseValue indicates the amount of a coinbase value does not
+	// match the expected value of the subsidy plus the sum of all fees.
+	ErrBadCoinbaseValue = ErrorKind("ErrBadCoinbaseValue")
+
+	// ErrBadCoinbaseFraudProof indicates that the fraud proof for a coinbase
+	// input was non-null.
+	ErrBadCoinbaseFraudProof = ErrorKind("ErrBadCoinbaseFraudProof")
+
+	// ErrBadCoinbaseAmountIn indicates that the AmountIn (=subsidy) for a
+	// coinbase input was incorrect.
+	ErrBadCoinbaseAmountIn = ErrorKind("ErrBadCoinbaseAmountIn")
+
+	// ErrBadStakebaseAmountIn indicates that the AmountIn (=subsidy) for a
+	// stakebase input was incorrect.
+	ErrBadStakebaseAmountIn = ErrorKind("ErrBadStakebaseAmountIn")
+
+	// ErrBadStakebaseScriptLen indicates the length of the signature script for
+	// a stakebase transaction is not within the valid range.
+	ErrBadStakebaseScriptLen = ErrorKind("ErrBadStakebaseScriptLen")
+
+	// ErrBadStakebaseScrVal indicates the signature script for a stakebase
+	// transaction was not set to the network consensus value.
+	ErrBadStakebaseScrVal = ErrorKind("ErrBadStakebaseScrVal")
+
+	// ErrScriptMalformed indicates a transaction script is malformed in some
+	// way.  For example, it might be longer than the maximum allowed length or
+	// fail to parse.
+	ErrScriptMalformed = ErrorKind("ErrScriptMalformed")
+
+	// ErrScriptValidation indicates the result of executing transaction script
+	// failed.  The error covers any failure when executing scripts such
+	// signature verification failures and execution past the end of the stack.
```

Comment:
Maybe I'm forgetting English but first sentence seems off. And such -> such as in the second.

Reply:
Modified.